packages/stream-metadata/src/opensea.ts

```diff
@@ -5,17 +5,123 @@ import { FastifyBaseLogger } from 'fastify'
 import { config } from './environment'
 import { spaceDapp } from './contract-utils'
 
-const getOpenSeaAPIUrl = (logger: FastifyBaseLogger, space: SpaceInfo) => {
+type GetNFTs = {
+	nfts: { identifier: string }[]
+	next?: string
+}
+
+const getAllMemberTokenIds = async (
```

Comment:
@miguel-nascimento does this function actually "get all member token ids"? do we need to talk to opensea to get a string of all our member token ids? isn't this an array where each element counts up from 0 to totalSupply?

Reply:
Thanks for the suggestion, you're right, we can get the members in the space and count 0..members.length

Reply:
members in the space !== the number of tokens. they are different and almost guaranteed to get out of sync

Reply:
Should that matter? We're just trying to reset all member nfts. Besides just like we both pointed out in the other comment, resetting all N tokens with N requests seems fragile. So hopefully none of this will be necessary and we can bulk-reset the entire contract with one request 🤞